copy-changelog.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs')
+const path = require('path')
+
+const rootDir = path.resolve(__dirname)
+const packagesDir = path.resolve(rootDir, 'packages')
+
+const rootChangelog = fs.readFileSync(path.resolve(rootDir, 'CHANGELOG.md'), 'utf8')
+
+const packages = fs.readdirSync(packagesDir).filter(package => !['.DS_Store', '@types'].includes(package))
```

Comment:
Didn't we want to copy it to only desktop and mobile? I mean the ones that actually end up on releases page

Reply:
We can do it in that way or have the same changelog in every package for consistency.

Reply:
I'd say let's copy it to only releasable packages (no-private)